docs/UserGuide.adoc

```diff
@@ -111,6 +111,32 @@ Deletes the 2nd person in the address book.
 `delete 1` +
 Deletes the 1st person in the results of the `find` command.
 
+== Updating tags of a person : `tag`
+
+Update tags on the specified person in the address book. +
+Format: `tag INDEX +TAG [-TAG ...]`
+
+****
+Updates tags of the person at the specified `INDEX`.
+The index refers to the index number shown in the most recent listing.
+****
+
+Examples:
+
+* `list` +
+`tag 2 -friend` +
+Removes the 'boyfriend' tag on the 2nd person in the address book.
+
+* `find Jack` +
+`tag 1 +nemesis +owesMeMoney` +
+Adds the 'nemesis' and 'owesMeMoney' tags to the 1st person in the results of the `find` command.
+
+* `find Tim` +
+`tag 1 +enemy -CS2103TGroupMate -boyFriend +exbf`
```

Comment:
You probably forgot to enter a newline here.

Reply:
Nice find! Thanks!